src/cmd/root.go

```diff
@@ -177,6 +178,13 @@ func rootHelp(cmd *cobra.Command, args []string) {
 }
 
 func rootRun(cmd *cobra.Command, args []string) error {
+	if len(args) == 0 {
+		rootFlags.assumeYes = true
+		rootFlags.enterShortcut = true
+
+		return enter(enterCmd, args)
```

Comment:
It would be better not to re-use the top-level entry points for a command, even when functionally one command includes other. eg., `enter()` doesn't actually call `run()` or `create(), even though it does include both those two commands.

Each of the top-level entry points, even if largely the same, are still subtly different (eg., when a container is created as part of `enter` we don't show the hint about how to use it, and `run` is a lot more strict about what to run than `enter`), and it feels icky to re-use the `cobra.Command` of a different command in this way (the `root` command isn't `enter`).

So, directly re-using the top-level entry point with a fake `cobra.Command` will only cause confusion. In this case, let's handle the top-level details separately and invoke `runCommand`.

We might notice some new patterns that are repeating between `root`, `enter` and `run`, and be able to shuffle or refactor the internal functions to better accommodate them, but the top-level functions are also not that big and the code in them is mostly catering to the actual differences in those commands.

Reply:
> `enter()` doesn't actually call run() or `create(), even though it does include both those two commands.

True but `enter()` provides extra functionality on top of `run()`. The root command was meant as an alias for `create` hence the call.

Still, you've got a point with this not being a nice way to go about it. This just makes my desire for clearly splitting the command-line code from the actual Toolbox functionality grow.